lib/handlers/api.js

```diff
@@ -111,11 +111,38 @@ class ApiHandler {
         this.outputList(availableLanguages, 'Id', req, res);
     }
 
+    filterCompilerProperties(list, selectedFields) {
+        return _.map(list, (compiler) => {
+            const filteredProps = {};
```

Comment:
Does `_.pick()` do what we need here? (total fly-by review! this is all awesome but just spotted a chance to use underscore :))

Reply:
I thought about pluck but that wouldn't. But just looked at pick and it looks like it would. Should probably be using that instead.